### PR TITLE
Bump version to 1.5.0

### PR DESCRIPTION
**Pull request by zhonghui12**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.5.0
+* Feature - Automatically re-create CloudWatch log groups and log streams after they are deleted(#95)
```
Comment (PettitWesley) [inline CHANGELOG.md]: no space in between deleted and parenthesis

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+* Feature - Add default fallback log group and stream names (#99)
+* Feature - Add support for special variables in log stream and group names (#108)
```
Comment (PettitWesley) [inline CHANGELOG.md]: Lets specifically say: ECS Metadata and UUID via special variables

Comment (PettitWesley) [reply CHANGELOG.md]: Users will look to the changelog to know what we added

Comment (zhonghui12) [reply CHANGELOG.md]: Got it. I will make it more specific

File: CHANGELOG.md (hunk 1 above)
Comment (PettitWesley) [reply CHANGELOG.md]: Also change to "if they are deleted", I like that better than "after"

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**